### PR TITLE
fix: Accordion release audit

### PR DESCRIPTION
**Pull request by reidbarber**
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/WAI/ARIA/apg/)

### CODE REVIEW

File: packages/react-aria/src/index.ts [hunk 1]
```diff
@@ -10,7 +10,6 @@
  * governing permissions and limitations under the License.
  */
 
-export {useAccordionItem, useAccordion} from '@react-aria/accordion';
```
Comment (snowystinger) [inline packages/react-aria/src/index.ts:13]: I don't think we can remove these, this would be breaking

File: packages/react-aria/src/index.ts [hunk 2]
```diff
@@ -49,7 +48,6 @@ export {useTooltip, useTooltipTrigger} from '@react-aria/tooltip';
 export {chain, mergeProps, useId, useObjectRef, RouterProvider} from '@react-aria/utils';
 export {VisuallyHidden, useVisuallyHidden} from '@react-aria/visually-hidden';
 
-export type {AriaAccordionProps, AccordionAria, AccordionItemAria, AccordionItemAriaProps} from '@react-aria/accordion';
```
Comment (snowystinger) [inline packages/react-aria/src/index.ts:52]: same

File: packages/react-aria-components/docs/Disclosure.mdx [hunk 3]
```diff
@@ -103,7 +103,7 @@ import {Disclosure, Button, DisclosurePanel, Heading} from 'react-aria-component
 
 ## Features
 
-Disclosures can be built with the [&lt;details&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) and [&lt;summary&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary) HTML element, but this can be difficult to style especially when adding adjacent interactive elements, like buttons, to the disclosure's heading. `Disclosure` helps achieve accessible disclosures implemented with the correct ARIA pattern while making custom styling easy. 
+Disclosures can be built with the [&lt;details&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) and [&lt;summary&gt;](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/summary) HTML elements, but this can be difficult to style, especially when adding adjacent interactive elements, like buttons, to the disclosure's heading. `Disclosure` helps achieve accessible disclosures implemented with the correct ARIA pattern while making custom styling easy.
```
Comment (snowystinger) [inline packages/react-aria-components/docs/Disclosure.mdx:106]: we should add accordion to the tags up above
`keywords: [disclosure, collapse, expand, aria, accordion]`

File: packages/react-aria/src/index.ts (hunk 1 above)
Comment (reidbarber) [reply packages/react-aria/src/index.ts:13]: I believe these were added after the last release.

Comment (snowystinger) [reply packages/react-aria/src/index.ts:13]: yep, we just got to it in our audit, these can be removed

**Review by snowystinger — APPROVED** [review]

**Review by devongovett — DISMISSED** [review]
One other thing I noticed is that react-aria-components currently has a dependency on `@react-aria/accordion` which is unused. We should remove that too.

File: packages/react-aria/src/index.ts (hunk 1 above)
Comment (devongovett) [inline packages/react-aria/src/index.ts:13]: We should remove this from the package.json too

**Review by devongovett — APPROVED** [review]

File: .storybook-s2/docs/Migrating.jsx [hunk 4]
```diff
@@ -34,6 +34,16 @@ export function Migrating() {
           <li className={style({font: 'body', marginY: 8})}>Update <Link href="https://react-spectrum.adobe.com/react-spectrum/styling.html#style-props">style props</Link> to use the <Link href="?path=/docs/style-macro--docs">style macro</Link> instead. See the 'Style props' section below</li>
         </ul>
 
+        <H3>Accordion</H3>
+        <ul className="sb-unstyled">
+          <li className={style({font: 'body', marginY: 8})}>Update <Code>Item</Code> to be <Code>Disclosure</Code>. <Code>Disclosure</Code> should now consist of two children: <Code>DisclosureHeader</Code> and <Code>DisclosurePanel</Code></li>
```
[the remark this answers is not in the document]
Comment (reidbarber) [reply .storybook-s2/docs/Migrating.jsx]: Good catch! I wrote this before we updated the API.

**Review by snowystinger — APPROVED** [review]

**Review by LFDanLu — APPROVED** [review]

**Pull request merged**